tests/network/vmi_multus.go

```diff
@@ -834,6 +867,29 @@ func cloudInitNetworkDataWithStaticIPsByDevice(deviceName, ipAddress string) str
 	return networkData
 }
 
+func cloudInitConfigDriveNetworkData(NICName, NADName, MACAddress, ipAddress, netmask string) string {
```

Comment:
please don't use first capital letter on passed params, it looks confusing in the method (looks like exported variable).

Reply:
Done